packages/common/container-definitions/src/error.ts

```diff
@@ -80,17 +80,14 @@ export interface ContainerWarning extends IErrorBase {
 /**
  * Represents errors raised on container.
  *
- * @see
+ * @remarks
  *
  * The following are commonly thrown error types, but `errorType` could be any string.
  *
- * - {@link @fluidframework/core-interfaces#ContainerErrorTypes}
+ * - {@link @fluidframework/core-interfaces#FluidErrorTypes}
  *
- * - {@link @fluidframework/driver-definitions#DriverErrorTypes}
+ * - {@link @fluidframework/driver-definitions#(DriverErrorTypes:variable)}
  *
- * - {@link @fluidframework/odsp-driver-definitions#OdspErrorTypes}
```

Comment:
Referencing client-specific error types from this layer violates our abstractions.